src/frontend/apps/standalone_site/src/App.tsx

```diff
@@ -14,6 +15,7 @@ import './App.css';
 const App = () => {
   const [currentTranslation, setCurrentTranslation] =
     useState<Record<string, string>>();
+  const queryClient = useMemo(() => new QueryClient(), []);
```

Comment:
Why not above App? You avoid useMemo like that.

Reply:
Well, I don't see any reason to declare it globaly rather than declaring it in the component exposing it